### ISSUE TITLE
AirPlay does not work on LG OLED (c1)

### ISSUE THREAD

**Issue by FMaz008** [futo-org/grayjay-android#399]
Using GrayJay v.201:

My cellphone, a Samsung A70, is on the same subnetwork as my TV.
My TV is wired, but for testing purpose, I also connected it to WiFi. (No change)

Upon clicking the Cast button in the app, the TV is detected. When accessing the AirPlay menu of the TV, it is instructing me to connect to the TV name, which matches what's listed in the casting menu.

When I click "start" to cast a youtube video via AirPlay, it spin indefinitely.

I tried to manually add the TV using it's wired IP, by default it adds it as a FC device. I'm not sure if my TV support that protocol, but it definitely support AirCast.

**Comment by xephos1one** [comment]
> Upon clicking the Cast button in the app, the TV is detected. When accessing the AirPlay menu of the TV, it is instructing me to connect to the TV name, which matches what's listed in the casting menu.
When I click "start" to cast a youtube video via AirPlay, it spin indefinitely.

Same behavior on my Sony running GoogleTV/Android 10.
The phone says "Connecting to device..." and the thing is spinning but nothing happens on the TV.
When trying with an iPhone (without Grayjay obviously) when selecting the TV via AirPlay it brought up a pairing code.

**Comment by Zvonimir-FUTO** [comment]
This issue is a bit stale. Is this still an issue with the latest version of Grayjay?

**Comment by FMaz008** [comment]
> This issue is a bit stale. Is this still an issue with the latest version of Grayjay?

Tried it just now with my LG Oled C1 and it is still not working. Any troubleshooting I can do for you?

**Comment by Zvonimir-FUTO** [comment]
> > This issue is a bit stale. Is this still an issue with the latest version of Grayjay?
> 
> Tried it just now with my LG Oled C1 and it is still not working. Any troubleshooting I can do for you?
> 
> App version 264 Trying with youtube videos

Sad to hear issue is still present. Could you also please upload logs?

**Comment by FMaz008** [comment]
I just did (verbose mode), I was not able to copy the UUID tho. Hopefully there is not too much.

I let the connection attempt go on for about 60 seconds before clicking stop

**Comment by Zvonimir-FUTO** [comment]
We picked up the logs and will have a look into it. Thank you for your patience.

**Comment by Airradda** [comment]
Support Airplay was dropped, correct? If so, then issue should probably be closed as not planned.

**Comment by Zvonimir-FUTO** [comment]
Yes, support was dropped. Forgot to close this one, thank you for the bump.
